packages/core/src/util/config.ts

```diff
@@ -102,4 +102,8 @@ export class Config {
   static getIheRequestsBucketName(): string | undefined {
     return getEnvVar("IHE_REQUESTS_BUCKET_NAME");
   }
+
+  static getIheParsedResponsesBucketName(): string {
+    return getEnvVarOrFail("IHE_PARSED_RESPONSES_BUCKET_NAME");
+  }
```

Comment:
Should we make this optional so we don't need two PRs?

The API will very likely be deployed before the infra finishes updating its task definition, so it will load without the env var... since this is used on stuff that we don't control (and it's quite busy), this will likely lead to runtime errors.

Reply:
@leite08 the variable is already in the lambda via this release https://github.com/metriport/metriport/pull/2502

I staged the release already.

Reply:
Yea, I didn't add it to the API, only the lambda -- so all the tests fail. Will make it optional.